Add the Language Introduction Video Slide Deck

praneesha opened the pull request:
## Purpose
Add the language introduction video slide deck.
> Fixes #

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

anupama-pathirage commented on learn-beta2.md:
```diff
@@ -10,7 +10,7 @@ redirect_from:
 ## Getting Started
 
-### Language Introduction Video
+### [Language Introduction Video](/learn/language-concepts/Ballerina_Swan_Lake_Presentation_Deck_V1.0.pdf)
```
Why are we adding a PDF for a Video topic?

praneesha replied:
Updated.

anupama-pathirage commented:
@praneesha To me this is bit confusing as we have many slide topics. As a new user to the page, these different slides, their descriptions, usage, purpose etc is not clear.

<img width="998" alt="image" src="https://user-images.githubusercontent.com/16300038/128680644-fee0f9ca-9d8e-4471-a2d5-e8de7ecd4b1b.png">

praneesha commented:
> @praneesha To me this is bit confusing as we have many slide topics. As a new user to the page, these different slides, their descriptions, usage, purpose etc is not clear.
> 
> <img alt="image" width="998" src="https://user-images.githubusercontent.com/16300038/128680644-fee0f9ca-9d8e-4471-a2d5-e8de7ecd4b1b.png">

It was planned during the design discussions to have 2 sets of videos and slides as "Language Introduction" and "Language Walkthrough".  Removed the "Language Introduction Slides" from the "Getting Started" section and moved the "Language Walkthrough Slides" under the "References" section since we have the  "Language Introduction Slides" also there. 

Will add the "Language Introduction Video' under the "Getting Started" section once we have it.

anupama-pathirage commented on learn-beta2.md:
```diff
@@ -46,6 +38,10 @@ A series of guides on various things you need to do during the Ballerina develop
 
 Presentation slides on the Ballerina language that anyone can use to talk about the language.
 
+### [Language Walkthrough Slides](http://localhost:4000/learn/language-concepts/Ballerina_Language_Presentation-2021-03-08.pdf)
```
Why localhost here? And it seems `Language Introduction Slides` and `Language Walkthrough Slides` both referring to the same .pdf. Why is that?

praneesha replied:
They were added just to complete the structure until we have 2 videos. Anyway, removed it and fixed the link.

anupama-pathirage commented on learn-beta2.md:
```diff
@@ -42,10 +30,14 @@ A series of guides on various things you need to do during the Ballerina develop
 
 Presentation slides on the Ballerina language that anyone can use to talk about the language.
 
+### [Language Walkthrough Slides](/learn/language-concepts/Ballerina_Language_Presentation-2021-03-08.pdf)
```
Language Walkthourgh video is having under `Getting Started`. It seems this is the slide set used for that video. So shouldn't we have that slides next to the video. Otherwise having the two forms of the same thing in two different sections doesn't look ok to me.

praneesha replied:
Updated.
